webpack/ForemanInventoryUpload/Components/IpsObfuscationSwitcher/__tests__/__snapshots__/IpsObfuscationSwitcherActions.test.js.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`IpsObfuscationSwitcher actions should handleToggle 1`] = `
+Array [
+  Array [
+    Object {
+      "payload": Object {
+        "ipsObfuscationEnabled": undefined,
```

Comment:
seem like the test should have been could with the `ipsObfuscationEnabled` as a parameter